packages/lambda-powertools-middleware-correlation-ids/event-sources/eventbridge.js

```diff
@@ -0,0 +1,35 @@
+const CorrelationIds = require('@dazn/lambda-powertools-correlation-ids')
+const consts = require('../consts')
+
+function isMatch (event) {
+  if (!event.hasOwnProperty('detail') ||
```

Comment:
It would have been nice if we had some more AWS specific props in the payload but understand this is probably the best for now

Reply:
You can publish custom events to EventBridge/CloudWatch Events as well as receiving vendor events (EventBridge only), and AFAIK these are the common fields that they all have